packages/block-library/src/image/image.js

```diff
@@ -173,7 +173,7 @@ export default function Image( {
 		! isContentLocked &&
 		! ( isWideAligned && isLargeViewport );
 	const imageSizeOptions = map(
-		filter( imageSizes, ( { slug } ) =>
+		imageSizes.filter( ( { slug } ) =>
```

Comment:
Defaults to an array:

https://github.com/WordPress/gutenberg/blob/a4040e40965b5430874413e5d5389ac889ad13b9/packages/block-editor/src/store/defaults.js#L133